### ISSUE TITLE
Display if build configuration is being loaded from YAML or UI

### ISSUE THREAD

**Issue by IlyaFinkelshteyn** [appveyor/ci#990]
It should stay in build trace too, for future investigations

**Comment by ericnewton76** [comment]
I would display this in the build log, and on the settings screen, if last build has that flag set (ie something like USING_APPVEYOR_YML=true)

Corollary to this would be that the appveyor.yml could be separate from the repo, loaded in parallel from the UI itself, and support at least download yml/upload yml functionality.  Just a suggestion.

**Comment by techtonik** [comment]
Upload appveyor.yml is supported - https://github.com/techtonik/appvey

**Comment by SPodjasek** [comment]
It would be very nice to see this in build console:
![image](https://user-images.githubusercontent.com/3293161/32688878-62622960-c6d9-11e7-9e3a-d5676300d466.png)

**Comment by IlyaFinkelshteyn** [comment]
We are working on the bigger feature to provide more audit audit data to the customers. This will be done as part of it.

**Comment by IlyaFinkelshteyn** [comment]
Closing in favor of https://github.com/appveyor/ci/issues/2140

**Comment by IlyaFinkelshteyn** [comment]
Closing in favor of https://github.com/appveyor/ci/issues/2140
